File: 2022/c_of_n.c
#include <math.h>
#include <stdio.h>

void main() {
	int n;

	scanf("%d", &n);

	int r = (int)sqrt(n);
	int count = 0;

	for (int a = 0; a < r; a++) {
		for (int b = 0; b < r; b++) {
			if (a * a + b * b < n) {
				count++;
			}
		}
	}

	printf("%d", count);
}

File: 2022/c_of_n.py
#!/bin/env python

import math

n = int(input())
count = 0

for a in range(0, math.ceil(math.sqrt(n))):
    for b in range(0, math.ceil(math.sqrt(n))):
        if a**2 + b**2 < n:
            count += 1

print(count)
